src/app/examples/examples-others.component.ts

```diff
@@ -39,4 +39,9 @@ export class ExemplesOthersComponent extends Examples {
                 break;
         }
     }
+    
+    changeDataAndValue31() {
+        this.data31 =  JSON.parse(JSON.stringify(data22));
+        this.value31m = ["heliotrope"] as never
+    }
```

Comment:
I feel like this test breaks the language switch test.

Reply:
you mean that the button is not translated?
I don't know french so ;)

but this test was kind of just to show the problem. You can remove it if you want?

I was thinking to really make a cypress test (that also runs at github) but that needs to be setup then completely first to use it for component testing 
(like ours now: https://github.com/Servoy/servoy-extra-components/blob/master/projects/servoyextracomponents/src/select2tokenizer/select2tokenizer.cy.ts that is a wrapper around the select2 component)

Reply:
made it a separate sample, needed to change a bit of typing (to be more precise what it really was so it was easier to work with later on)

Reply:
i hope this fix can be released quickly because we can work around it by hacking our own component but i don't really like todo this (you can skip the whole sample if you want)